app/build.gradle

```diff
@@ -67,11 +67,11 @@ dependencies {
 
     //debug implementation
     //debugImplementation "androidx.compose.ui:ui-tooling:$compose_version"
-    implementation "org.jetbrains.kotlin:kotlin-stdlib:1.1.60"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib:1.6.10"
```

Comment:
In order to successfully debug, I checked the contents of the error report to solve the "caused by: org.gradle.internal.resolve.moduleversionnotfoundexception: could not find com.squareup.retrofit2: retrofit2: 2.7.1
”Wait for a question, I'm based on“ https://stackoverflow.com/questions/52215153/unable-to-resolve-dependency-for-retrofit2 ”The version has been revised according to your comments